tests/e2e/tools/lib/covers.js

```diff
@@ -0,0 +1,74 @@
+const {expect} = require('@playwright/test');
+
+const TAG_NAMES = ['Climate', 'Oceans','renewables', 'Consumption'];
+const PAGE_NAMES = ['Vestibulum leo libero', 'Consectetur adipiscing elit', 'Vestibulum placerat'];
+const POST_TYPES = ['Publication','Press Release'];
+
+async function addCoversBlock(page, style = '') {
+  // Add Covers block.
+  await page.locator('.block-editor-block-list__layout').click();
+  await page.locator('p.is-selected.wp-block-paragraph').fill('/planet-4-covers');
+  await page.keyboard.press('Enter');
+
+  // Select the style if needed.
+  if (style) {
+    const stylePicker = page.locator('.block-editor-block-styles__variants');
+    //CSS selector needs single word,hence remove word after space(eg Take Action=>Take).
+    await stylePicker.locator(`button[aria-label^=${style.split(" ")[0]}]`).click();
+  }
+
+  if (style === 'Take Action') {
```

Comment:
Please kindly ignore, this piece of code(if condition), it belongs to other [PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1064/files#diff-5008d74bb8ff2d12c0bcc5a657baf88cd1e6b7ed8221c4f2c03dbbd48bf1aed4R19) which is still in review.